### ISSUE TITLE
Update to Unicode v9

### ISSUE THREAD

**Issue by mmainstreet** [rodrigopolo/jqueryemoji#4]
A lot of new Emoji came with v9 (see http://www.unicode.org/emoji/charts/emoji-released.html).
It would be nice if you add these :)

**Comment by praveenscience** [comment]
Yes, it would be nice to have the latest one like [Male Technologist](http://emojipedia.org/male-technologist/) emojis...

**Comment by mmainstreet** [comment]
I made a version with all new emoji:
[unicode_v9.zip](https://github.com/rodrigopolo/jqueryemoji/files/984572/unicode_v9.zip)
